mappings/net/minecraft/client/render/OverlayTexture.mapping

```diff
@@ -1,6 +1,11 @@
 CLASS net/minecraft/class_4608 net/minecraft/client/render/OverlayTexture
 	FIELD field_21013 texture Lnet/minecraft/class_1043;
+	FIELD field_21444 DEFAULT_UV I
 	METHOD method_23209 setupOverlayColor ()V
 	METHOD method_23210 getU (F)I
 	METHOD method_23212 getV (Z)I
 	METHOD method_23213 teardownOverlayColor ()V
+	METHOD method_23624 encodeUv (FZ)I
+	METHOD method_23625 encodeUv (II)I
```

Comment:
`packUv`, if this is just packing two numbers into a single int (`u << 16 + v`)?